main.tf

```diff
@@ -288,9 +296,10 @@ module "alb_http_sg" {
 
   name        = "${var.name}-alb-http"
   vpc_id      = local.vpc_id
-  description = "Security group with HTTP ports open for specific IPv4 CIDR block (or everybody), egress ports are all world open"
+  description = "Security group with HTTP ports open for specific IPv4 and IPv6CIDR block (or everybody), egress ports are all world open"
 
-  ingress_cidr_blocks = sort(compact(concat(var.allow_github_webhooks ? var.github_webhooks_cidr_blocks : [], var.alb_ingress_cidr_blocks)))
+  ingress_cidr_blocks = local.alb_ingress_ipv4_cidr_blocks
+  ingress_ipv6_cidr_blocks = local.alb_ingress_ipv6_cidr_blocks
```

Comment:
Please introduce another variable like `var.alb_ingress_ipv6_cidr_blocks` and use it instead of this unnecessary magic :)

Reply:
sure, will do 👍

Reply:
@antonbabenko so... the thing is the magic in the locals takes into account blocks from 2 lists `var.allow_github_webhooks` and `var.alb_ingress_cidr_blocks`.

Do you think there should be ipv6 list of the GitHub web hooks blocks accordingly?

Reply:
`var.github_webhooks_cidr_blocks` (right?) contains IPv4 addresses so no need to do anything with it.

Reply:
yes, sorry the `var.github_webhooks_cidr_blocks`.

At the moment it can only accept IPv4 (since it's passed to the `ingress_cidr_blocks` of the security group rule.

Since now GitHub started to use IPv6 for web hooks (`2a0a:a440::/29` and `2606:50c0::/32`), how do we pass these addresses to the security group rule as well? Those can't go in the same `var.github_webhooks_cidr_blocks` variable as that one is limited to IPv4 only.

Reply:
Then let's introduce another variable for GitHub IPs like `var.github_webhooks_ipv6_cidr_blocks` and have them separated.